x-pack/filebeat/docs/inputs/input-httpjson.asciidoc

```diff
@@ -1133,6 +1133,90 @@ request_url using file_name as 'file_2': \https://example.com/services/data/v1.0
 +
 Collect and make events from response in any format supported by httpjson for all calls.
 
++
+[[chain-step-replace_with]]
+[float]
+==== `chain[].step.replace_with`
+
+The `replace_with: "pattern,value"` clause is used to replace a pattern defined in the `request.url` with a given value. The value can be hard coded or can be 
```

Comment:
Agreed, but one concern I have is, in our case we are not replacing any string, but specific strings with a certain structure. Every old_value string in our case needs to start with **$.** which make it more similar to a pattern rather than a normal string. Maybe we could add a **NOTE** saying traditional regex patterns are not supported ?

Reply:
I'm thinking "fixed pattern" would probably cover the distinction.